Specification/BrAPI-Schema/BrAPI-Phenotyping/Scale.json

```diff
@@ -72,12 +72,9 @@
                     ]
                 },
                 "validValues": {
-                    "relationshipType": "one-to-many",
+                    "relationshipType": "one-to-one",
```

Comment:
reverse association is still many-to-one

Reply:
I think it is 1 to 1, always. The 'ValidValues' entity is always associated with a Scale. It is a Composition association. It only exists with a single Scale object and no other.